packages/rs-dapi-client/src/dapi_client.rs

```diff
@@ -114,12 +132,17 @@ impl DapiRequestExecutor for DapiClient {
         TransportError: Mockable,
     {
         // Join settings of different sources to get final version of the settings for this execution:
-        let applied_settings = self
```

Comment:
Hate this applied settings. Should something like transport request settings since they go to transport request